scrapers/FootFetishDaily.yml

```diff
@@ -0,0 +1,147 @@
+name: "Foot Fetish Daily"
+sceneByURL:
+  - action: scrapeXPath
+    url:
+      - footfetishdaily.com/update/
+    scraper: sceneScraper
+sceneByFragment:
+    # this works if the fileNames are "as-is" f/ the studio
+    # or at least if you keep the sceneID at the front. ex: 234_sceneName.mp4
+    action: scrapeXPath
+    scraper: sceneScraper
+    queryURL: "https://footfetishdaily.com/update/{filename}/"
+    queryURLReplace:
+      filename:
+        - regex: "^(\\d+).*$"
+          with: $1
+galleryByURL:
+  - action: scrapeXPath
+    url: 
+      - footfetishdaily.com/update/
+    scraper: galleryScraper
+galleryByFragment:
+    # galleryIDs (until ~2020) are the same as sceneId, after: they are sceneId+1. 
+    # either way (like scenes) galleryID must be @start of filename. ex: 234_someName.zip
+    action: scrapeXPath
+    scraper: galleryScraper
+    queryURL: "https://footfetishdaily.com/update/{filename}/"
+    queryURLReplace:
+      filename:
+        - regex: "^(\\d+).*$"
+          with: $1
+performerByURL:
+  - action: scrapeXPath
+    url:
+      # REQ: (/ID/)[name:optional] # ex /230/Dahlia_Denyle
+      - footfetishdaily.com/model/
+      - footfetishdaily.com/guest/models/
+    scraper: performerScraper
+xPathScrapers:
+  sceneScraper:
+    scene:
+      Title: &title
+        selector: //main//h1
+        postProcess:
+          - replace:
+              - regex: "( Remastered| Photoset)"
+                with: ""
+      Details: &details
+        selector: //main//p
+        postProcess:
+          - replace:
+              - regex: "(\\*Trailer not available for this update.|Join Now!)"
+                with: ""
+      Date: &date
+        selector: //main//h3
+        postProcess:
+          - replace:
+              - regex: "Release Date:"
+                with: ""
+      Studio: &studio
+        Name: //nav//h1
+      Performers: &performers
+        Name: //h2/a/text()
+        URL: //h2/a/@src
+      Image: &image //video/@poster | //video-js/@poster | //img[@class="img-fluid"]/@src
+      URL: &url
+        selector: //link[contains(@rel,'canonical')]/@href
+        postProcess:
+          - replace:
+              - regex: ^.(.*).$
+                with: $1
+  galleryScraper:
+    gallery:
+      Title: *title
+      Details: *details
+      Date: *date
+      Studio: *studio
+      Performers: *performers
+      URL: *url
+  performerScraper:
+    performer:
+      Name: //main//h1/text()
+      Ethnicity: //label[contains(text(),"Ethnicity")]/../..//p/text()
+      Measurements: //label[contains(text(),"Bra Size")]/../..//p/text()
+      Image: 
+        selector: //img[@class="rounded-circle"]/@src
+      Height: 
+        selector: //label[contains(text(),"Height")]/../..//p/text()
+        postProcess:
+          - feetToCm: true
```

Comment:
The site does not keep a consinstenf format for the height.
It seems to be a mix  eg `5'5` in `https://footfetishdaily.com/model/976/Jasmine_Wilde`
`165 cm` in `https://footfetishdaily.com/model/983/Luna_Haze`
I am not sure what is the best option here....
Leaving the feetToCm will get us the correct values for the performer with measurements in feet but will give huge numbers for the rest
Postprocessing and removing the `cm` will leave us with correct height for the performers with cm but leave the feet unconverted
feetToCm
```
165 cm -> 5029
5'5 -> 165
```
No feetToCm
```
165 cm  ->  165
5'5 -> 5'5
```

Reply:
your right - i've commented out the height chunk and noted they are inconsistent on this one